Project/demo/src/main/java/com/example/demo/model/authenticationController.java

```diff
@@ -0,0 +1,37 @@
+    package com.example.demo.model;
+
+    import org.springframework.beans.factory.annotation.Autowired;
+    import org.springframework.stereotype.Controller;
+    import org.springframework.ui.Model;
+    import org.springframework.web.bind.annotation.GetMapping;
+    import org.springframework.web.bind.annotation.PostMapping;
+
+
+    @Controller
+    public class authenticationController
```

Comment:
By convention class name always starts with capital letter. Object names and variable names start with small letter.